flipper-plugin/src/data/ClassInfo.tsx

```diff
@@ -1,11 +1,11 @@
 export interface ClassInfo {
-  name: string;
+  classSignature: string;
   superClassName: string;
   properties: PropertyInfo[];
 }
 
 export interface PropertyInfo {
-  name: string;
+  signature: string;
   type: string;
   valueType: string;
   debuggable: boolean;
```

Comment:
These model classes should be exported from kotlin source.

Reply:
Done on [Migrate to kotlin-defined PropertyInfo](https://github.com/kitakkun/back-in-time-plugin/pull/143/commits/9c1614ba35c018b348d39374c225522f69c01e18)

Reply:
ClassInfo has no corresponding definition on Kotlin source.